src/history/history-entry.enity.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import {
+  Column,
+  Entity,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { User } from '../users/user.entity';
+import { Note } from '../notes/note.entity';
+
+@Entity()
+export class HistoryEntry {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
```

Comment:
Why do these need their own id? Shouldn't the composite key with `noteId` and `userId` suffice?

Reply:
Yeah this is probably not necessary.

Reply:
done